### DISCUSSION TITLE
Load higher quality videos.

### DISCUSSION THREAD

**Discussion by MrTheenD** [rodit/SnapMod#185] [Ideas & Feature Suggestions]
I've been using 10.48 version of Snapchat for quite a while on my second smartphone and keep using the latest versions of Snapchat on my main one. It's very noticeable the huge video quality drop between the two versions (~4MB vs. ~700KB per video). Would it be within the scope of this app to integrate a feature that allows users to force Snapchat to fetch the high quality snaps instead of the crappy ones?

**Comment by rodit** [comment]
Are these the same exact same snaps (i.e. from the same stories) being downloaded on both devices?

**Reply by MrTheenD** [reply]
> Are these the same exact same snaps (i.e. from the same stories) being downloaded on both devices?

Yes. With same network conditions and same account. Also, I tried installing 10.48 version on my main device ended up with the same result, video quality was just better on older version.

I uploaded raw video files saved from both versions in this [drive link](https://drive.google.com/drive/folders/1Rc4bgvrHCMqjCQ9ZTr14bnWiAcMqhv0Z?usp=sharing). I used another Xposed module to save those snaps on older version but It still would load HQ videos without the module.

Though, it's not a big deal for me as I still continue using the old version.



> The simplest and easiest solution is to simply update Snapchat on the 2nd device. I thought Snapchat had already ceased supporting 10.48 anyway; but if not it's coming soon regardless. In which case there will be no option other than to update.

It's not supported but it's operational in my specific use case, which is loading snap stories. Other than that, it's pretty much broken.

**Comment by clublek** [comment]
Hope this can be implemented, as all images are saving in WebP format, instead of true JPG.
